cmd/soroban-cli/src/commands/config/mod.rs

```diff
@@ -63,15 +63,32 @@ pub struct Args {
     pub ledger_file: ledger_file::Args,
 
     #[arg(long, alias = "source", env = "SOROBAN_ACCOUNT")]
-    /// Account that signs the final transaction.
-    /// S...          a seceret key
-    /// alice         an identity
-    /// 'kite urban.  a seed phrase
-    /// DEFAULT       Is the key generated with `identity generate --seed 0000000000000000
+    /// Account that signs the final transaction. Alias `source`
+    ///
+    /// SC36BWNUOCZA...  a secret key
+    ///
+    /// alice            an identity
+    ///
+    /// "kite urban .."  a seed phrase
+    ///
+    /// If not provided the default is the seed phrase generated by  
+    /// `identity generate --default-seed`
+    ///
+    /// Examples:
+    ///
+    ///     --source alice
+    ///
+    ///     --source "kite urban.."
+    ///
+    ///     --source SC36BWNUOCZAO7DMEJNNKFV6BOTPJP7IG5PSHLUOLT6DZFRU3D3XGIXW
     pub source_account: Option<String>,
 
     #[arg(long)]
     /// If using a seed phrase, which hd path to use, e.g. `m/44'/148'/{hd_path}`
+    ///
+    /// Example: `--hd-path 1`
+    ///
+    /// Default is `0`
```

Comment:
If we like the idea of making these single (long) lines, we could replace this with:

```suggestion
    /// If using a seed phrase, which hierarchical deterministic path to use, e.g. `m/44'/148'/{hd_path}`. Example: `--hd-path 1`. Default: `0`
```

Another question: is that `e.g.` value accurate? Seems like it's missing a couple segments, based on others I've seen and the examples in [this article](https://medium.com/myetherwallet/hd-wallets-and-derivation-paths-explained-865a643c7bf2).